src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/UsersController.cs

```diff
@@ -25,6 +25,7 @@ public ActionResult<UserDetailsViewModel> MyDetails()
                     Email = User.Claims.FirstOrDefault(x => x.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress")?.Value,
                     Name = User.Claims.FirstOrDefault(x => x.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname")?.Value + " " + User.Claims.FirstOrDefault(x => x.Type == "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname")?.Value,
                     Permissions = new string[] { "team lead" },
+                    Claims = User.Claims
```

Comment:
There's nothing sensitive in these Claims that we'd be sending to the front end, is there?

Reply:
Could be yep, im hoping i can inspect it quickly then revert this commit once we have the required details